runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -327,27 +327,27 @@ class VM_JFRConstantPoolTypes {
 
 	/* Event Types */
 	J9Pool *_executionSampleTable;
-	UDATA _executionSampleCount;
+	U_32 _executionSampleCount;
 	J9Pool *_threadStartTable;
-	UDATA _threadStartCount;
+	U_32 _threadStartCount;
 	J9Pool *_threadEndTable;
-	UDATA _threadEndCount;
+	U_32 _threadEndCount;
 	J9Pool *_threadSleepTable;
-	UDATA _threadSleepCount;
+	U_32 _threadSleepCount;
 	J9Pool *_monitorWaitTable;
-	UDATA _monitorWaitCount;
+	U_32 _monitorWaitCount;
 	J9Pool *_threadParkTable;
-	UDATA _threadParkCount;
+	U_32 _threadParkCount;
 	J9Pool *_cpuLoadTable;
-	UDATA _cpuLoadCount;
+	U_32 _cpuLoadCount;
 	J9Pool *_threadCPULoadTable;
-	UDATA _threadCPULoadCount;
+	U_32 _threadCPULoadCount;
 	J9Pool *_classLoadingStatisticsTable;
-	UDATA _classLoadingStatisticsCount;
+	U_32 _classLoadingStatisticsCount;
 	J9Pool *_threadContextSwitchRateTable;
 	U_32 _threadContextSwitchRateCount;
 	J9Pool *_threadStatisticsTable;
-	UDATA _threadStatisticsCount;
+	U_32 _threadStatisticsCount;
```

Comment:
It's not clear to me that we want to change the types of these fields. Please explain the motivation here.

Reply:
I think using `UDATA` is more consistent with data-type usage elsewhere in the VM, but, of course the getters need to be consistent, so they must return `UDATA` as well.